types/react/test/elementAttributes.tsx

```diff
@@ -26,5 +26,6 @@ const testCases = [
     <span autoCorrect="on" />,
     <span translate="no" />,
     <span translate="yes" />,
-    <svg><image crossOrigin="anonymous"/></svg>,
+    <svg><image crossOrigin="anonymous" /></svg>,
+    <input value={['one', 'two'] as ReadonlyArray<string>} />
```

Comment:
This here is a problem. While I understand that you want to be correct, it would be mostly a nuisance for developers.

Reply:
I'm not quite sure what exactly would be a problem, could you elaborate?
Just to clarify, there are no breaking changes, this would also be a valid expresson:

```jsx
<input value={['one', 'two']} />
```

Reply:
Could you add a test to make sure?

I guess `<input value={['one', 'two'] as ReadonlyArray<string>} />` was rejected before?

Reply:
Added tests, yes, you are right, previously it wouldn't accept ReadonlyArray.